test/parallel/test-assert-checktag.js

```diff
@@ -45,7 +45,8 @@ function re(literals, ...values) {
   }
   assert.doesNotThrow(() => assert.deepEqual(fakeGlobal, global));
   // Message will be truncated anyway, don't validate
-  assert.throws(() => assert.deepStrictEqual(fakeGlobal, global));
+  assert.throws(() => assert.deepStrictEqual(fakeGlobal, global),
+                /^AssertionError: /);
```

Comment:
Will do. I actually tried to put `AssertionError` first but realized it's not globally available. Your comment made me look closer and I saw that it is exported by the `assert` module.